src/Fixer/PhpUnit/PhpUnitMockFixer.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\PhpUnit;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Indicator\PhpUnitTestCaseIndicator;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+final class PhpUnitMockFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Usages of `->getMockWithoutInvokingTheOriginalConstructor` method MUST be replaced by `->createMock` method.',
```

Comment:
Why not supporting `getMock()` as well?

Reply:
see opening ticket

Reply:
> This PR doesn't contain getMock replacement, as there is no 1-1 replacing method

`getMock()` could be replaced by `createMock()` when only one argument is provided. This is a common case (I think).